src/timeline/TimelineEventsVisualizer.scss

```diff
@@ -2,7 +2,48 @@
 
 #timeline {
   margin: 20px;
-  text-align: left;
+  text-align: left;  
```

Comment:
[JA1]: I don't think we need any of this styling anymore since we're sticking with our HoverItem for this approach.